CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+cmake_minimum_required(VERSION 3.10)
+
+#-------------------------------------------------------------------------------
+# BASIC SETUP
+#-------------------------------------------------------------------------------
+
+set(prog_name    keystone-sdk)
+set(src_dir      ${CMAKE_CURRENT_LIST_DIR})
+set(scripts_dir  ${src_dir}/scripts)
+
+if (OUTPUT_DIR)
+    set(out_dir      ${OUTPUT_DIR})
+else()
+    set(out_dir      ${CMAKE_CURRENT_LIST_DIR})
+endif()
+
+#-------------------------------------------------------------------------------
+# Program and flags
+#-------------------------------------------------------------------------------
+
+include(macros.cmake)
+
+if (NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE    "Debug")
+else()
+    if ((NOT CMAKE_BUILD_TYPE STREQUAL "Debug") AND (NOT CMAKE_BUILD_TYPE STREQUAL "Release"))
+    message(FATAL_ERROR "CMAKE_BUILD_TYPE must either be Denug or Release instead of ${CMAKE_BUILD_TYPE}")
```

Comment:
typo: Denug -> Debug

Reply:
Hi! Thanks for reviewing the my suggestion.

I fixed my problem in the newest commit and fixed typo error.
My reason for moving from make to Cmake that: CMake is very usefull for small or large project. CMake is cross-platform and mordern. CMake syntax is clear than Makefile. It's wonderful when you maintain and extend project.